test/ex_doc/language/erlang_test.exs

```diff
@@ -15,12 +15,12 @@ defmodule ExDoc.Language.ErlangTest do
 
     test "current module", c do
       assert autolink_edoc("{@link erlang_foo}", c, current_module: :erlang_foo) ==
-               ~s|<a href="erlang_foo.html#content"><code>erlang_foo</code></a>|
```

Comment:
Should `#content` anchor be kept? From quick repo search its role seems to be to point at the top. In that case, should it be added to all relative hrefs?

Reply:
@josevalim in case this got lost in my other comments, could you please confirm that it's OK for this hashtag to be dropped?

Reply:
The test: https://github.com/elixir-lang/ex_doc/blob/d1b0950d3f313573477f3fdf08c1364448788779/test/ex_doc/language/erlang_test.exs#L16-L19

Reply:
Yes, 100% ok.